lib/convenient_service/rspec/matchers/custom/import.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+module ConvenientService
+  module RSpec
+    module Matchers
+      module Custom
+        class Import
+          ##
+          # @param method_name [Symbol, String]
+          # @param scope [Symbol]
+          # @param from [Class, Module]
```

Comment:
Looks like `DependencyContainers` have incomplete behavior.

Could you please, implement a logic that forbids to use of `export` in classes?

Thanks.

Reply:
@marian13 Check the following PR #61 &#127881; &#127881; &#127881;